html/code-journal.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>BLOG</title>
+  <link href="../stylesheets/styles.css"   rel="stylesheet">
+  <link href="../stylesheets/normalize.css" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <h1><a class="home-button" href="index.html"> Marisol Lopez </a></h1>
+
+    <nav>
+      <ul class="nav-bar">
+        <li class="nav-link"><a href="about.html">ABOUT</a></li>
+        <li class="nav-link"><a href="portfolio.html">PROJECTS</a></li>
+        <li class="nav-link"><a href="code-journal.html">BLOG</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <section class ="main-section">
+      <h2>BLOG</h2>
+      <div class="blog">
+        <p>
+          Topgallant square-rigged transom Privateer yardarm fire ship boom chandler wherry Blimey. Lanyard quarterdeck to go on account tender hogshead lugsail fire ship Spanish Main draft run a rig. Fire ship bowsprit Arr coffer topsail sheet fire in the hole gabion keel Barbary Coast.        </p>
+        </div>
+        <div class="blog">
+          <p>
+            Me bilge water red ensign case shot mizzenmast pirate black spot tender ho grapple. Interloper chandler pillage no prey, no pay lookout haul wind belay Barbary Coast boatswain mizzen. Avast wherry jolly boat maroon boatswain clap of thunder cable lookout poop deck ahoy.
+
+          </p>
+        </div>
+
+      </section>
+
+
+    </main>
+
```

Comment:
Is the use of section appropriate here? I might also use a <small> element to wrap the copyright. (https://developers.facebook.com/docs/instant-articles/reference/footer)